### PR TITLE
Add store endpoint for owasm studio

### LINKED ISSUE #324: Separate `store` from current `request` endpoint follow new specification
- Split current `request` endpoint to 2 endpoint `store` and `request` used in different product (studio and scan)
- This part just add new endpoint and use code from current endpoint

Endpoint specification
input
```
{
    "code": "006....",
    "name": "Script test"
}
```

output
```
{
     "txhash": "0d23123..."
}
```

### PR DESCRIPTION
**Pull request by taobun**
fixed #324 

Store endpoint to store code with code and name return txHash and codeHash

```
{
    "code": "0006...",
    "name": "New code"
}
```

return
```
{
    "txHash": "57DF90E375B1679033DA1F83451FA721A4A75FE8BDFB199B42083490E0E1036D",
    "codeHash": "2D3CF926D1D11E11EBE917DAD59056BCEBE5D17AFFD53676FA941168B3FB24CD"
}
```

### CODE REVIEW

**Review by prin-r — DISMISSED** [review]

**Review by prin-r — DISMISSED** [review]

**Comment by taobun** [comment]
@sorawit @smiled0g Please review.

**Review by prin-r — DISMISSED** [review]

**Review by sorawit — DISMISSED** [review]

File: chain/cmtx/utils.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ func NewTxBuilder(txEncoder sdk.TxEncoder) authtypes.TxBuilder {
 	fee, _ := sdk.ParseCoins("")
 	gasPrices, _ := sdk.ParseDecCoins("")
 	// TODO: Remove hard code gas limit and gas adjustment
-	return authtypes.NewTxBuilder(txEncoder, 0, 0, 10000000, 1, false, "bandchain", "", fee, gasPrices)
+	return authtypes.NewTxBuilder(txEncoder, 0, 0, 20000000, 1, false, "bandchain", "", fee, gasPrices)
```
Comment (sorawit) [inline chain/cmtx/utils.go:42]: How big can the code be?

Comment (taobun) [reply chain/cmtx/utils.go:42]: Current logic.rs (include every lib) use 10342549 from induction It will handle 2 bigger size of code.

**Review by sorawit — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Review by prin-r — APPROVED** [review]

**Pull request merged**